x-pack/plugins/enterprise_search/public/applications/shared/kibana/kibana_logic.test.ts

```diff
@@ -33,6 +33,12 @@ describe('KibanaLogic', () => {
       expect(KibanaLogic.values.config).toEqual({});
     });
 
+    it('gracefully handles disabled security', () => {
+      mountKibanaLogic({ ...mockKibanaValues, security: undefined } as any);
+
```

Comment:
(comment attached to random code)
The subnav disappears on Schema error page:

https://user-images.githubusercontent.com/11838280/122598065-b3545700-d042-11eb-83b9-0ef20aa31ee3.mp4

Let me know if you need help with reproducing this case!

Reply:
This is because of Workplace Search's schema routing. Note the URL pattern:

- sources/{id}/schema
- sources/{id}/**schema_errors**/{indexJobId}

The schema_errors URL doesn't register as a valid route present on the nav, so it disappears.

If you have no objections to changing the URL, I would suggest either `sources/{id}/schema/errors/{indexJobId}` or `sources/{id}/schema/{indexJobId}`

BTW, App Search also changed its schema_errors route and it works quite well for us:

<img width="1050" alt="" src="https://user-images.githubusercontent.com/549407/122599172-c5260200-d022-11eb-9098-9cb58c60b7a5.png">

Reply:
@scottybollinger Do you see any problems with changing routes? I don't and I think we could make them the same as App Search routes.

Reply:
Here's an example of this working (+ with breadcrumbs):

![schema](https://user-images.githubusercontent.com/549407/122604589-0ae6c880-d02b-11eb-98f3-6a27426c43ce.gif)

I pushed up an example/proposed commit here: https://github.com/elastic/kibana/pull/102592/commits/b6a62b63ea65fa6a62a2d8486b204a418cc72190

But happy to either revert it if you don't want the URL change, or tweak the URL if @scottybollinger has feedback

Reply:
The commit looks good and works perfectly! If Scotty has a reason not to change the routes, we can revert it in this or future PR.